ee/docker-compose.ch.yml

```diff
@@ -16,10 +17,11 @@ services:
     clickhouse:
         # KEEP CLICKHOUSE-SERVER VERSION IN SYNC WITH
         # https://github.com/PostHog/charts-clickhouse/blob/main/charts/posthog/templates/clickhouse_instance.yaml#L88
-        image: yandex/clickhouse-server:21.6.5
+        image: yandex/clickhouse-server:21.8.5.7
```

Comment:
Note: This won't work due to https://github.com/PostHog/posthog/issues/5684

Reply:
This PR is only to test to see if it is viable as a solution (why it's a draft). So far it's not viable as it doesn't work! Going to keep poking at this later to see if I can get it working and debug what is going on.